go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/anchore/stereoscope
 
-go 1.19
+go 1.20
```

Comment:
there are some trixy generics bugs in 1.20 that I'm not certain were back ported, we might want to set this as 1.21 (don't consider this as a must-do comment, but a consideration that we need to validate)